runtime/compiler/control/JITClientCompilationThread.cpp

```diff
@@ -1137,6 +1137,18 @@ handleServerMessage(JITServer::ClientStream *client, TR_J9VM *fe, JITServer::Mes
          }
          break;
 #endif // J9VM_OPT_OPENJDK_METHODHANDLE
+      case MessageType::VM_isStable:
+         {
+         auto recv = client->getRecvData<J9Class *, int>();
+         J9Class *fieldClass = std::get<0>(recv);
+         int cpIndex = std::get<1>(recv);
+
+         bool isStable = false;
+         if (fieldClass)
```

Comment:
Since we don't send a message when fieldClass is NULL we should use an assert here rather than an `if` test